src/django_idom/components.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from django.contrib.staticfiles.finders import find
+from idom import component, html
+
+from django_idom.config import IDOM_CACHE
+
+
+@component
+def static_css(static_path: str):
```

Comment:
This brings up a bit of a debate.

I mentioned in the past that I prefer if we follow PEP8 styling for component names instead of React styling. This is mostly to avoid pissing off users that use Python style linters.

Whatever we decide on this will likely decide what we do in the future for any prefab components.

Reply:
Going to assume we're pushing forward with PascalCase so I'm going to mark this as resolved.

Reply:
While trying to convert Conreq component names to PascalCase, the pylint warnings started to bother me so I re-opened this.

Reply:
@rmorshea Let's discuss this before PR merge.

This is important to hash out now since this is our first production-capable prefab component.

Reply:
I think for the sake of (1) alone, we should probably use snake case going forward.

Reply:
I also realized that making them camel case makes it seem like component functions can be used in type annotations, which they cannot.